support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/heading.tsx

```diff
@@ -251,6 +264,26 @@ function Heading({
 		);
 	}
 
+	if (isPaperOrGuardianWeekly) {
+		const getPrintTitle = (productOption?: string) => {
+			switch (productOption) {
+				case 'Sixday':
+					return 'Six day';
+				case 'Everyday':
+					return 'Every day';
+				default:
+					return productOption;
+			}
```

Comment:
Possibly one for a future refactor, but there are quite a few places where we map `ratePlanKey` or `productKey` to user friendly labels, I wonder whether we could put this lable in product catalog so we can have it defined in one place and read it from there?

Reply:
Yes, I think we need an `activeRatePlanKey` type defined from the `productCatalog`, similar to what Tom did with 
 the `activeProductKey` type.  As you say, possibly outside of the scope of this work (its used as a string throughout the code-base currently). Want me to  create a seperate trello card?

Reply:
@paul-daniel-dempsey yes if you can create a card that would be great, it looks like a useful thing to have typed. Thanks! 🙏